features/tabbed-content.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Tabbed Content
+  In order to have confidence in my site's tabbed content
+  As an anonymous user
+  I want to verify that the tabbed content functions properly
+
+  Background:
+    Given I am an anonymous user
+    When I am at "/tabbed-content"
+    Then the ".uds-tabbed-panels" element should exist
+
+  @api @javascript @anonymous
+  Scenario: Verify users can view tabbed content
+    Given the "#nav-tab" element exists
+    Then I should see that the "#nav-tab ~ .scroll-control-prev" element exists
+    And the "#nav-tab ~ .scroll-control-next" element exists
+    And the "#nav-tabContent" element exists
+
+  @api @javascript @anonymous
+  Scenario: Verify the first item is highlighted
+    Given the "#nav-tab-1-tab" element exists
+    Then only one element should have the selector "#nav-tab-1-tab"
+    And the "#nav-tab-1-tab" element should have the class ".active"
+    # NOTE: This definition easily provides false positives
+    And I should see that "border-bottom" with "8px solid #8c1d40" is in ".uds-tabbed-panels .nav-tabs .nav-link.active" class
+
+  @api @javascript @anonymous
+  Scenario: Verify the items content is properly visible
+    Given the "#nav-tab-1-tab" element exists
+    And the "#nav-tab-1" element exists
+    Then the "#nav-tab-1" element should have the classes ".active.show"
+
+  @api @javascript @anonymous
+  Scenario: Verify selecting a new item highlights the item and displays the content
+    Given the "#nav-tab-2-tab" element exists
+    Then the "#nav-tab-1-tab" element should exist
+    And the "#nav-tab-1-tab" element should have the class ".active"
+    And the "#nav-tab-2-tab" element should not have the class ".active"
+    When I click the element "#nav-tab-2-tab"
+    Then the "#nav-tab-2-tab" element should have the class ".active"
+    And the "#nav-tab-1-tab" element should not have the class ".active"
+    And the "#nav-tab-2" element should have the classes ".active.show"
```

Comment:
I've run a this test about 10 times and it succeeded once, but failed here the other 9. Perhaps a short pause is needed do all of the updates propagate?

<img width="809" alt="Screenshot 2024-04-30 at 2 32 33 PM" src="https://github.com/ASUWebPlatforms/webspark-ci/assets/212131/c7924f2c-f780-40b9-be89-c3261d6b02a9">

Reply:
@mlsamuelson added in some pauses. Note that the GTM scripts can interfere with some tests if the full screen modal pops up during the test.